modules/homefinder/homefinder.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * HomeFinder Module
+ *
+ * @package RE-PRO
+ */
+
+/* Exit if accessed directly. */
+if ( ! defined( 'ABSPATH' ) ) { exit; }
```

Comment:
__[code_sniffer]__
* Expected 0 spaces after opening bracket; 1 found
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found
* Newline required after opening brace
* Closing brace must be on a line by itself

<!-- Comment by [SideCI](https://sideci.com) -->